scripts/ios/download_generator

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Can you add the .sh extension? we use that in every script and i think it makes it easier to distinguish that it's a bash script

Reply:
Yeah this is actually intentional... On OSX you can embed scripts to be executable.. so this makes this basically a double-click and run option rather than the classic terminal ->./run_script.sh.

<img width="558" alt="screen shot 2018-08-12 at 2 49 12 pm" src="https://user-images.githubusercontent.com/830748/43998706-7bc8ee32-9e3f-11e8-8c30-a1e29be1bbe4.png">

The OS reads the first line and know how to run it. I can post-fix add ".command" if its ambiguous. Let me know what you think

Reply:
can't .sh be double clicked as well?